doc/conseil.sql

```diff
@@ -33,6 +33,8 @@ COMMENT ON EXTENSION plpgsql IS 'PL/pgSQL procedural language';
 -- Name: truncate_tables(character varying); Type: FUNCTION; Schema: public; Owner: -
 --
 
+DROP FUNCTION IF EXISTS public.truncate_tables(username character varying);
```

Comment:
Out of curiosity: did you add this manually?

AFAIK this script is not supposed to be run on existing instances. I'm not sure if we may prefer for it to fail if that happens.

Reply:
Yes, I added this manually because I needed to remove this function manually each time I created a new database.